gencred/cmd/main.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api/latest"
+	gencred "k8s.io/test-infra/gencred/pkg"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// defaultContextName is the default context name.
+	defaultContextName = "build"
+	// defaultConfigFileName is the default kubeconfig filename.
+	defaultConfigFileName = "/dev/stdout"
+)
+
+// options are the available command-line flags.
+type options struct {
+	context        string
+	name           string
+	output         string
+	certificate    bool
+	serviceaccount bool
+	overwrite      bool
+}
+
+// parseFlags parses the command-line flags.
+func (o *options) parseFlags() {
+	// --context
```

Comment:
This seems a) unnecessary and b) better inlined with StringVar

Reply:
Will remove.